From 0929e21b7a65842969c58c9ec4566fa8e1b89624 Mon Sep 17 00:00:00 2001
From: Varun Saxena <varunsaxena@apache.org>
Date: Tue, 9 Aug 2016 23:35:44 +0530
Subject: [PATCH 1920/2748] MAPREDUCE-6750. Fix
 TestHSAdminServer#testRefreshSuperUserGroups
 (Kihwal Lee via Varun Saxena)

(cherry picked from commit c4b77aed7e756dd463fd88141054ea0e3acd6396)

Change-Id: I7fc94da087ae1ca51c40f05542c524d7135daf18
---
 .../mapreduce/v2/hs/server/TestHSAdminServer.java  |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/server/TestHSAdminServer.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/server/TestHSAdminServer.java
index d831347..1eb1d1c 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/server/TestHSAdminServer.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/server/TestHSAdminServer.java
@@ -187,7 +187,8 @@ public void testRefreshSuperUserGroups() throws Exception {
     when(ugi.getRealUser()).thenReturn(superUser);
     when(superUser.getShortUserName()).thenReturn("superuser");
     when(superUser.getUserName()).thenReturn("superuser");
-    when(ugi.getGroupNames()).thenReturn(new String[] { "group3" });
+    when(ugi.getGroups())
+        .thenReturn(Arrays.asList(new String[] { "group3" }));
     when(ugi.getUserName()).thenReturn("regularUser");
 
     // Set super user groups not to include groups of regularUser
-- 
1.7.9.5

